src/components/cta/cta.tsx

```diff
@@ -24,7 +24,44 @@ export class CtaComponent {
   // optional message prop
   @Prop() public message?: string;
 
+  /**
+   * Set href attribute if you want the output to be link else button will be created
+   */
+  @Prop() public href?: string;
+  /**
+   * Set default text content if not passed while link tag is generated
+   */
+  @Prop() public link: string = 'link';
+  /**
+   * Set default text content if not passed while button tag is generated
+   */
+  @Prop() public button: string = 'button';
+
   public render() {
-    return <div>{this.message}</div>;
+    let props = {};
+    /**
+     * Choose if href is passed in tag then a tag is generated else button tag is generated
+     */
+    const Wrapper: any = this.href ? 'a' : 'button';
+    /**
+     * Pass inner content depending upon type of tag created
+     */
+    const innertext: any = this.href ? this.link : this.button;
+
+    /**
+     * Set href props
+     */
+
+    if (this.href) {
+      props = {
+        href: this.href
+      };
+    }
+    return (
+      <Wrapper {...props}>
+        {innertext}
+        <div>{this.message} </div>
```

Comment:
Div in side a button or Link??? This is semantically wrong.

Reply:
I can use span instead.